css/glowingbear.css

```diff
@@ -326,6 +326,7 @@ tr.bufferline {
 td.time {
     padding: 1px 5px 1px 1px;
     vertical-align: top;
+    text-align: right;
```

Comment:
having a 12-hour time locale stops the hour field returned from `Date.toLocaleTimeString()` from being zero-padded, so if we have something like:

```
9:59 pm [text]
10:00 pm [text]
```

This change makes the above turn into:


```
 9:59 pm [text]
10:00 pm [text]
```